### PR TITLE
Deprecate --watch

### PR DESCRIPTION
**Pull request by asottile**
See https://github.com/dahlia/libsass-python/issues/155#issuecomment-235489107

### CODE REVIEW

File: sassc.py [hunk 1]
```diff
@@ -40,7 +40,12 @@
    Watch file for changes.  Requires the second argument (output CSS
    filename).
 
+   Note that ``--watch`` does not understand imports.  Due to this, the
```
Comment (dahlia) [inline sassc.py]: It would be better if we use [`.. note::`](http://docutils.sourceforge.net/docs/ref/rst/directives.html#note) here.

Comment (asottile) [inline sassc.py]: Indeed! Fixed :)

**Comment by dahlia** [comment]
Looks good to me!

**Comment by asottile** [comment]
Sweet, thoughts on #157? It's why this build is failing

**Pull request merged**